externalTestApi/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+import java.net.URI
+
+plugins {
+    `java-library`
+}
+
+repositories {
+    val github = ivy {
+        url = URI("https://github.com/")
+        patternLayout {
+            artifact("/[organisation]/[module]/releases/download/[revision]/[classifier].[ext]")
+        }
+        metadataSources { artifact() }
+    }
+
+    exclusiveContent {
+        forRepositories(github)
+        filter { includeGroup("pantsbuild") }
+    }
+}
+
+val externalSystemTestApi: Configuration by configurations.creating
+
+fun externalSystemTestApiJar() = zipTree(externalSystemTestApi.singleFile).matching {
+    include("external-system-test-api.jar")
+}
+
+dependencies {
+    externalSystemTestApi("pantsbuild:intellij-pants-plugin:external-system-test-api-v211.7142.45:external-system-test-api@zip")
```

Comment:
Version can be parametrized

Reply:
Done

Reply:
Actually, should this be merged with the IJ version or do we want a possibility to have different test-api.jar and IJ versions?

Reply:
I think we can, as we tend to avoid bumping the test jar version if it is not necessary. We may end up with bumping a minior version of ij for some reason and leaving test api jar as is.